rocAL/rocAL/source/meta_data/external_source_label_reader.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright (c) 2019 - 2022 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Please change the copyright 2023, applicable to all such instances in the new files added